data/campaigns/fri03.wmf/scripting/mission_thread.lua

```diff
@@ -11,6 +11,43 @@ function insert_soldiers(tbl, what)
    end
 end
 
+function port_space_blocked(f)
+   for i,field in pairs({
+      f,
+      f.ln,
+      f.ln.ln,
+      f.rn,
+      f.brn,
+      f.brn.rn,
+      f.bln,
+      f.bln.ln,
+      f.brn.brn,
+      f.brn.bln,
+      f.tln,
+      f.tln.ln,
+      f.trn,
+      f.trn.rn,
+      f.trn.trn,
+      f.trn.tln,
+      f.tln.tln,
+   }) do
+      if field.owner ~= nil then return true end
```

Comment:
this is true as soon as the expedition places my port which made me loose my first testing attempt. needs to be checked for differenmt owner then p1 as well